zeppelin-interpreter/src/main/java/org/apache/zeppelin/display/AngularObject.java

```diff
@@ -81,6 +90,7 @@ public String getName() {
 
   /**
    * Set noteId
+   *
```

Comment:
How meaningful are this newlines in context of the task of implementing new NotebookRepo ?